sdk/core/azure-core-amqp/inc/azure/core/amqp/management.hpp

```diff
@@ -159,12 +167,15 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
      *
      * @returns a ManagementOperationResult which includes the high level result of the operation,
      * the HTTP response status code, the status description, and the response message.
+     *
+     * @remark The messageToSend is intentionally passed by value because the ExecuteOperation needs
+     * to modify the message to add the required properties for the management operation.
      */
     ManagementOperationResult ExecuteOperation(
         std::string const& operationToPerform,
         std::string const& typeOfOperation,
         std::string const& locales,
-        Azure::Core::Amqp::Models::AmqpMessage const& messageToSend,
+        Models::AmqpMessage messageToSend,
```

Comment:
does this need to be a copy?

Reply:
The simple answer is: Yes.

In response to your previous question, I left a comment in the code saying:
```cpp
     * @remark The messageToSend is intentionally passed by value because the ExecuteOperation needs
     * to modify the message to add the required properties for the management operation.
```

If you have suggestions on how I can improve the comment to make this clearer, I would appreciate them.